api/environments/job_handler.go

```diff
@@ -137,6 +166,38 @@ func (eh EnvironmentHandler) getBatches(appName, envName, jobComponentName strin
 	return eh.getScheduledBatchSummaryList(radixBatches), nil
 }
 
+// StopBatch Stop batch by name
+func (eh EnvironmentHandler) StopBatch(appName, envName, jobComponentName, batchName string) error {
+	batch, err := eh.getRadixBatch(appName, envName, jobComponentName, batchName, kube.RadixBatchTypeBatch)
+	if err != nil {
+		return err
+	}
+
+	if batch.Status.Condition.Type != radixv1.BatchConditionTypeActive && batch.Status.Condition.Type != radixv1.BatchConditionTypeWaiting {
```

Comment:
A RadixBatch that has not been processed by radix-operator will have an empty string ("") for Type. A user should be allowed to stop these batches

Reply:
fixed